omod/src/main/webapp/resources/scripts/sync2.conflict.js

```diff
@@ -0,0 +1,73 @@
+function conflictResolution() {
+
+    jq.getJSON('/' + OPENMRS_CONTEXT_PATH + '/sync2/AuditDetails/conflictResolution.action',
```

Comment:
why do you use jq and JQuery interchangeably?

Reply:
Good catch, fixed!